guides/source/active_record_querying.md

```diff
@@ -1267,7 +1267,7 @@ This is because it is ambiguous whether they should appear on the parent record,
 Scopes
 ------
 
-Scoping allows you to specify commonly-used queries which can be referenced as method calls on the association objects or models. With these scopes, you can use every method previously covered such as `where`, `joins` and `includes`. All scope methods will return an `ActiveRecord::Relation` object which will allow for further methods (such as other scopes) to be called on it.
+Scoping allows you to specify commonly-used queries which can be referenced as method calls on the association objects or models. With these scopes, you can use every method previously covered such as `where`, `joins` and `includes`. All scope bodies must return an `ActiveRecord::Relation` or `nil` to allow for further methods (such as other scopes) to be called on it.
```

Comment:
A scope will never return `nil` and if it did then this documentation would be wrong because you would not be able to chain another scope on a `nil` result.